daft/daft/__init__.pyi

```diff
@@ -1221,6 +1221,11 @@ def binary_length(expr: PyExpr) -> PyExpr: ...
 def binary_concat(left: PyExpr, right: PyExpr) -> PyExpr: ...
 def binary_slice(expr: PyExpr, start: PyExpr, length: PyExpr | None = None) -> PyExpr: ...
 
+# ---
+# dashboard namespace
+# ---
+def launch(static_assets_path: str): ...
```

Comment:
Nit: should this be `launch_dashboard`?

Reply:
Sure, can change the name. I used launch since I thought `dashboard.launch` would look clean.

Reply:
Whoops, realized you meant the `.pyi` binding name. I renamed the one inside of `__init__.pyi` to `launch_dashboard` and kept the one inside of `daft/dashboard` the same.